### ISSUE TITLE
wk1 aug_epic story_13 task: Review the following Modeling to Learn resources/guides to inform a Power BI MVP

### ISSUE THREAD

**Issue by anazariz** [lzim/teampsd#2112]
[MS Teams Link 1](https://teams.microsoft.com/l/message/19:1dea54c6db60416e8985ba6bf7bb7883@thread.skype/1624910554576?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1624910554576&teamName=teampsd_vha&channelName=quant_workgroup&createdTime=1624910554576)

[MS Teams Link 2](https://teams.microsoft.com/l/message/19:1dea54c6db60416e8985ba6bf7bb7883@thread.skype/1624911135618?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1624910554576&teamName=teampsd_vha&channelName=quant_workgroup&createdTime=1624911135618)

Review the following Modeling to Learn resources/guides to inform a Power BI MVP, starting with the Clinic Selection and SPReferrals tabs, with the least amount of total team rework across training & documentation.
 
Current State: Review 5 resources for how Modeling to Learn learners and facilitators use the Data UI (MTL Red part 1; Blue session 2) to produce a Team Data Table (MTL Red part 2; Blue session 3):
 	
[ ] 1. Review the MTL Blue 1.7 release videos (note: We're on MTL 2.0 release, but don't have updated videos) for session 2 & session 3:
	Session 2 - https://bcove.video/2PGS6vo
	Session 3 - https://bcove.video/38Buyk2

[ ] 2. Data UI cheatsheet for learners - https://github.com/lzim/mtl/blob/master/red/part1/part1_learner/mtl_how_data_cheatsheet.pdf
	 
[ ] 3. MTL Blue Sessions 2 & 3 SEE Learner Guides & MTL Red Part 1 & 2 SEE Learner Guides (these guides cover the same material across the MTL Blue and Red programs at a descriptive level for learners).
	https://github.com/lzim/mtl/blob/master/red/part1/part1_learner/mtl_red_part_1_see.md
	https://github.com/lzim/mtl/blob/master/red/part2/part2_learner/mtl_red_part_2_see.md
	 
[ ] 4. Data UI Facilitator 1-pager for facilitators -
	https://github.com/lzim/mtl/blob/master/red/part1/part1_facilitator/mtl_how_data_facilitator_one_pager.pdf
	 
[ ] 5. MTL Blue Sessions 2 & 3 SAY Facilitator Guides & MTL Red Part 1 & 2 SAY Facilitator Guides (these guides cover the same material across the MTL Blue and Red programs at a detail level for facilitators).
	https://github.com/lzim/mtl/blob/master/red/part1/part1_facilitator/mtl_red_part_1_say.md
	https://github.com/lzim/mtl/blob/master/red/part2/part2_facilitator/mtl_red_part_2_say.md
	 
There are 5 resources to review this week.

Please 

a) bring questions that you have 
b) related to developing wk1 jul_epic user-centered hypotheses for the Power BI MVP 
c) to this Thursday's 7/1 quant_operations workgroup meeting.

**NOTE - The 7/1 date was impacted by the SharePoint Office Hours meeting and follow up. On Friday, I implemented the sync points as  instructed by the SharePoint team and ran the most critical Facility files as notes in the [](url) [quant_operations Teams chat](https://teams.microsoft.com/l/message/19:1dea54c6db60416e8985ba6bf7bb7883@thread.skype/1625258223744?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&groupId=1db500d5-0d01-4254-af42-ad3f78bafacd&parentMessageId=1625193373961&teamName=teampsd_vha&channelName=quant_workgroup&createdTime=1625258223744)** .

**Comment by anazariz** [comment]
Reviewed:
- [ ] Review the MTL Blue 1.7 release videos
- [ ] Data UI cheatsheet for learners
- [ ] MTL Blue Sessions 2 & 3 SEE Learner Guides & MTL Red Part 1 & 2 SEE Learner Guides
- [ ] Data UI Facilitator 1-pager for facilitators
- [ ] MTL Blue Sessions 2 & 3 SAY Facilitator Guides & MTL Red Part 1 & 2 SAY Facilitator Guides

**Comment by anazariz** [comment]
**ROM Estimate start date(7/12 - 2 days on ops):** 

**Common Tasks:** 

- [ ] Figure out the provisioning of workspaces, etc. **1 wk**   A 

- [ ] RLS (Role Level Security) and AD (Active Directory) group design 1 wk Refresh Mechanism  **1 wk**    B 

- [ ] Data Models **1 wk** C 

- [ ] Caching mechanism **1 wk** B  

- [ ] Separation between Data UI and Facility from a design perspective **1 wk**  C 

- [ ] Reports/Dashboards/Workspaces overall design **1 wk** 

**Total = 4 wks** 

**Individual Tasks:** 

- [ ] Design/Develop Data UI **2 wks** (reassessed) 

- [ ] Test/Fix/Deploy Data UI **1 wk** 

- [ ] Design/Develop Facility **1 wk**  (some work was already done for MVP) 

- [ ] Test/Fix/Deploy Facility **1 wk** 

**Total = 5 wks** 

**Float:** 

1 wk 

ROM total = 10 weeks (serially – some task may be done in parallel) : July (1), Aug (4), Sep(4), Oct (1)

**Comment by lzim** [comment]
@anazariz 

While you may be prioritizing bugs #2115, #2126 & #2127 today, I want to remind you of the request at **quant_operations** on 7/15 to provide at least 1-2 bullets of what each of the tasks in the ROM entails, so that team members can begin to support this story.

For example, 

"**Figure out the provisioning of workspaces, etc.**" 1 wk A
"**RLS (Role Level Security) and AD (Active Directory) group design 1 wk Refresh Mechanism.** 1 wk B

Are vague for a team member to take action on as written.

But, @dkngenda @staceypark @lijenn and I can take action on better specified tasks.
For example, Stacey, Jenn & David, as you can see we can set up a **[Deployment Pipeline** for our PTSD_OMHO Power BI dashboard](https://app.powerbigov.us/pipelines) and there are many resources, office hours and our meetings with PBI SMEs this week for efficient supports to help us to do this.

![image.png](https://images.zenhubusercontent.com/5e4ac859b8781e56a5b7d6a1/00092d5d-688b-4b11-b428-2edbd3375e17)

**Comment by staceypark** [comment]
Cross Ref: #1444

**Comment by staceypark** [comment]
@jamesmrollins  I believe you've already reviewed the resources listed above, which is the task for this card.
Other work related to Power BI is tracked via #1961
